lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -597,29 +610,31 @@ impl Renderer {
 pub struct HardcodedLayers {
     /// A special layer used to store shapes not attached to any layer. This layer will not be
     /// rendered. You should not need to use it directly.
-    pub DETACHED:           Layer,
-    pub root:               Layer,
-    pub viz:                Layer,
-    pub below_main:         Layer,
-    pub main:               Layer,
-    pub port:               Layer,
-    pub port_selection:     Layer,
-    pub label:              Layer,
-    pub port_hover:         Layer,
-    pub above_nodes:        Layer,
-    pub above_nodes_text:   Layer,
+    pub DETACHED: Layer,
+    pub root: Layer,
+    pub viz: Layer,
+    pub below_main: Layer,
+    pub main: Layer,
+    pub port: Layer,
+    pub port_selection: Layer,
+    pub label: Layer,
+    pub port_hover: Layer,
+    pub above_nodes: Layer,
+    pub above_nodes_text: Layer,
     /// `panel` layer contains all panels with fixed position (not moving with the panned scene)
     /// like status bar, breadcrumbs or similar.
-    pub panel_background:   Layer,
-    pub panel:              Layer,
-    pub panel_text:         Layer,
-    pub node_searcher:      Layer,
+    pub panel_background_rect_level_0: RectLayerPartition,
+    pub panel_background_rect_level_1: RectLayerPartition,
+    pub panel_background: Layer,
```

Comment:
The docs were for `panel_background` field before. Also add docs for rect_level_0 as it's very unclear what is the purpose of those.

Reply:
I think the original comment had become unclear with previous changes---It is meant to refer to the group of `panel_` layers.

Reply:
I will update the comment to more clearly refer to the group of layers and partitions.